input/pagecontent/index.md

```diff
@@ -1,15 +1,86 @@
-### Introduction
 This IG provides guidance for applying security labels in FHIR. Security labels are used in access control systems governing the collection, access, use, and disclosure of the target information to which they are assigned, as required by applicable organizational, jurisdictional, or personal policies related to privacy, security, and trust.
 
-The FHIR model has supported security labels from the start with a dedicated element on every resources (regardless of the type of resource), enabling security label processing in a unified method independent of resource type.
+The syntactic and semantic rules of the [HL7 Healthcare Privacy and Security Classification System (HCS), Release 1 (HCS)](http://www.hl7.org/implement/standards/product_brief.cfm?product_id=345) provide a conceptual framework for constructing security labels in order to convey computable and interoperable privacy, security, and trust policies. One of the goals of this IG is to provide guidance on how to map and model HCS concepts in FHIR.
+
+From the outset, FHIR has supported security labels with a dedicated optional element ([`Resource.meta.security`](https://build.fhir.org/resource-definitions.html#Meta.security)) that can appear on any resource (regardless of resource type), therefore, enabling recording and processing of security labels in a unified manner. 
+### Terminology Caveats
+In FHIR, a security label is defined as a simple value of type [`Coding`](https://build.fhir.org/datatypes.html#Coding) which is composed of (among other details), a _code_ and the identifier of the _system_ in which the code is defined, together with a human readable _display_ name for the _code_. For example, the following data structure represents a FHIR security label for _restricted_ confidentiality as defined by HL7 terminology: 
+
```

Comment:
use of italics for .code, .system, and .display -- You should have these indicted as code with back quote.